src/Core/Shop/LogoUploader.php

```diff
@@ -107,14 +108,21 @@ public function update($fieldName, $logoPrefix, array $files = [])
             }
 
             $fileExtension = ($fieldName == 'PS_STORES_ICON') ? '.gif' : '.jpg';
+            if (in_array($files[$fieldName]['type'], ['image/svg+xml', 'image/svg'])) {
```

Comment:
Why not storing `['image/svg+xml', 'image/svg']` in a const and using a method to avoid duplicate check.
I see you have the same condition twice, not sure it's a good idea to keep both, but using `isSvgFileType($files[$fieldName]['type'])` will be easier to use and prevent error if a new mime type is added or if it needs to be changed

Reply:
@PierreRambaud I've commited your suggestion